ui/src/pages/deployment-details/deployment-details-info.tsx

```diff
@@ -83,7 +83,7 @@ export const DeploymentDetailsInfo = ({
         <FormSection title={translate(STRING.FIELD_LABEL_SOURCE_IMAGES)}>
           <FormRow>
             <InputValue
-              label="Data source"
+              label={translate(STRING.FIELD_LABEL_DATA_SOURCE)}
```

Comment:
@annavik The other select fields for Site & Device have a placeholder "Pick a value", but Data Source does not. I can't see why, any ideas?

Reply:
This is because if project has no data sources registered, it's not possible to "pick a value". Then we instead disable the control with label "Not available". Does it make sense? No problem to tweak this logic, let me know :)

If project has a storage registered:
<img width="750" alt="Screenshot 2024-03-18 at 22 25 07" src="https://github.com/RolnickLab/ami-platform/assets/11680517/1cc8f14f-4dbb-4537-99fd-16a8856bca47">

If project has no storage registered:
<img width="743" alt="Screenshot 2024-03-18 at 22 28 13" src="https://github.com/RolnickLab/ami-platform/assets/11680517/7846bbb8-179b-4892-9f1a-b415c345da5e">